lms/templates/courseware/program_marketing.html

```diff
@@ -338,11 +338,21 @@ <h2>${_('Instructors')}</h2>
         <h2>${_('Frequently Asked Questions')}</h2>
       </div>
       % for faq in faqs:
+        <%
+          faq_hash = hash(faq['question'])
+          is_truncated = len(faq['answer']) > 100
+        %>
         <div class="col-4 faq">
           <div class="question">
             ${faq['question']}
           </div>
-          <div class="answer">
+          <div id="preview-answer-${faq_hash}">
+            ${HTML(faq['answer'][:100])}
+            % if is_truncated:
```

Comment:
Would be better to not truncate based on length of characters but instead based on available space. For example, we should show three rows of text and if the text doesn't fit in three rows then show the "see more" link. Otherwise, on a wide screen seems wrong to truncate on 100 characters if there is room to show more characters in the allotted space

Reply:
I originally did this height-based, but there is no way to tell if content is going to be > X lines. Truncating by height makes more sense visually, but truncating it by line length, was what I saw suggested due to the dynamic "See more". This is how it's done on most sites. The 100 was semi-arbitrary, we could do a longer length if it helps.